dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -241,8 +241,8 @@ public void createServiceDuplicate() throws Exception {
         assertNotNull(service);
         try {
             client.addService(serviceRepo, "admin@admin.com", installationId);
-        } catch (Exception ex) {
-
+        } catch (ApiException ex) {
+            assertEquals("Should have error code 418", 418, ex.getCode());
```

Comment:
Should make a constant somewhere in common akin to https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.html#LENGTH_REQUIRED to track this custom http code.